chsdi/models/vector/bafu.py

```diff
@@ -2390,3 +2390,25 @@ class LandesforstinventarSchutzwaldregion(Base, LandesforstinventarRegion, Vecto
     __bodId__ = 'ch.bafu.landesforstinventar-schutzwaldregionen'
 
 register(LandesforstinventarSchutzwaldregion.__bodId__, LandesforstinventarSchutzwaldregion)
+
+
+class AmphibienwanderungVerkehrskonflikte(Base, Vector):
+    __tablename__ = 'amphibienwanderung_verkehrskonflikte'
+    __table_args__ = ({'schema': 'fauna', 'autoload': False})
+    __bodId__ = 'ch.bafu.amphibienwanderung-verkehrskonflikte'
+    __template__ = 'templates/htmlpopup/amphibienwanderung-verkehrskonflikte.mako'
+    __queryable_attributes__ = ['nr_datenbank', 'name']
```

Comment:
please dont ask this feature anymore, it wont be supported in our future api anymore.
@mariokeusen is that ok with you?

Reply:
how does this relate to the searchable attributes from the DIF?

Reply:
there is no relation between that and the fulltext search in sphinx. this line is for the configuration/activation of the query search tool which will be deprecated soon.

![image](https://user-images.githubusercontent.com/5286659/116871353-6a704c80-ac14-11eb-9237-4d73692c1ed4.png)

searchable attributes in sphinx are only indicating if theres a fulltext search or not.

Reply:
sorry, my bad. I removed it in accordance with Mario.